packages/app/src/cli/utilities/developer-platform-client/shopify-developers-client/graphql/active-app-release.ts

```diff
@@ -38,12 +40,15 @@ export interface AppModule {
   gid: string
   uid: string
   handle: string
-  config: object
+  config: {
+    [key: string]: string | number | boolean | string[]
```

Comment:
This is my quick attempt at a JSON type, but I think we probably need something better here down the road.